packages/core/src/util/ng_dev_mode.ts

```diff
@@ -115,7 +115,7 @@ export function initNgDevMode(): boolean {
     if (typeof ngDevMode !== 'object') {
       ngDevModeResetPerfCounters();
     }
-    return !!ngDevMode;
+    return typeof ngDevMode !== 'undefined';
```

Comment:
```suggestion
    return typeof ngDevMode !== 'undefined' && !ngDevMode;
```

I think this is more correct. Consider if `ngDevMode === 'false'` In that case function should return `false` not `true`